clients/node/examples/6-AuthorizeToChannel.ts

```diff
@@ -0,0 +1,74 @@
+import { AccessRights, IdentityClient, ChannelClient } from 'iota-is-sdk';
+
+import { defaultConfig } from './configuration';
+
+// In this example we will use two instances of the ChannelClient() both will authenticate a different user.
+const ownerClient = new ChannelClient(defaultConfig);
+const userClient = new ChannelClient(defaultConfig);
+const identity = new IdentityClient(defaultConfig);
+
+async function authorizeOthersToChannel() {
+  // Creating a channel owner who creates the channel and a channel user who will be authorized to read the channel
+  console.log('Creating user...');
+  const channelOwner = await identity.create('Owner');
+  const channelUser = await identity.create('User');
+
+  // We will use two instances of the channel api client. One is getting authorized by the owner and the other one by the user.
+  await ownerClient.authenticate(channelOwner.doc.id, channelOwner.key.secret);
+  await userClient.authenticate(channelUser.doc.id, channelUser.key.secret);
+
+  // The owner creates a channel where he/she want to publish data of type 'example-data'.
+  const { channelAddress } = await ownerClient.create({
+    topics: [{ type: 'example-data', source: 'example-creator' }]
+  });
+
+  console.log('Writing to channel...');
+  // Writing data to the channel as the channel owner.
+  await ownerClient.write(channelAddress, {
+    payload: { log: `This is log file 1` }
+  });
+
+  // This attempt to read the channel will fail because the channel user is no authorized to read the channel.
+  try {
+    const channelData = await userClient.read(channelAddress);
+  } catch (error: any) {
+    console.error('Error: ', error?.response?.data?.error);
+  }
+
+  // Request subscription to the channel as the user. The returned subscriptionLink can be used to authorize the user to the channel.
+  const { subscriptionLink } = await userClient.requestSubscription(channelAddress, {
+    accessRights: AccessRights.ReadAndWrite
+  });
+
+  console.log("Subscription Link:", subscriptionLink)
+
+  // Find subscriptions to the channel that are not already authorized.
+  const subscriptions = await ownerClient.findAllSubscriptions(channelAddress, false);
+
+  console.log("Subscriptions Found:", subscriptions);
+
+  const unauthorizedSubscriptions = subscriptions.filter(subscription => !subscription.isAuthorized)
```

Comment:
Normally the second parameter in `findAllSubscriptions(channelAddress, **false**` should only return authorized subscriptions. So this line will not be needed